<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>581</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="set">
     <attribute name="title">
      <string>set</string>
     </attribute>
     <widget class="QLabel" name="set_size_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>尺寸：</string>
      </property>
     </widget>
     <widget class="QLabel" name="set_num_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>张数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="set_content_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>内容：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="set_size">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>150</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>250</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>300</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>350</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="set_num">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="set_content">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>301</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="set_start">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>开始生成</string>
      </property>
     </widget>
     <widget class="QLabel" name="set_size_px">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>67</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>px</string>
      </property>
     </widget>
     <widget class="QLabel" name="set_path_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>保存路径：</string>
      </property>
     </widget>
     <widget class="QLabel" name="set_path">
      <property name="geometry">
       <rect>
        <x>116</x>
        <y>146</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="set_path_click">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>选择保存路径</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="show">
     <attribute name="title">
      <string>show</string>
     </attribute>
     <widget class="writeQRCode" name="show_qrcode">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="show_stop">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
     <widget class="QLabel" name="show_plain_text">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="shoe_cipher_text">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="show_dis">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="show_num">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="finish">
     <attribute name="title">
      <string>finish</string>
     </attribute>
     <widget class="QLabel" name="finish_num">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="finish_goback">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>返回首页</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>二维码生成器</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>writeQRCode</class>
   <extends>QLabel</extends>
   <header location="global">writeQRCode.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
